packages/pds/tests/views/notifications.test.ts

```diff
@@ -36,7 +36,7 @@ describe('pds notification views', () => {
       { headers: sc.getHeaders(alice) },
     )
 
-    expect(notifCount.data.count).toBe(12)
+    expect(notifCount.data.count).toBe(10)
```

Comment:
kinda a pain that these numbers keep changing. maybe we could have the seed client track them?

(not for this PR, just throwing out a thought)